wasm/src/lib.rs

```diff
@@ -28,6 +28,31 @@ pub fn git_hash() -> JsValue {
     JsValue::from_str(git_hash)
 }
 
+#[wasm_bindgen]
+pub fn provide_text_document_content(name: &str) -> JsValue {
```

Comment:
```suggestion
pub fn get_library_source_content(name: &str) -> JsValue {
```

Suggest name to parallel my uri scheme name suggestion.